packages/terra-site/src/examples/arrange/ArrangeAlignment.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Arrange from 'terra-arrange';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import _ from 'lodash';
```

Comment:
Lodash isn't declared in the package.json. I assume it's getting pulled in transitively, could you add it to the package for site?

Reply:
https://github.com/cerner/terra-core/pull/821/commits/7d270f071a4ca1d17ce4f6445ecc82aafcc709db